bundles/model/org.eclipse.smarthome.model.core/src/main/java/org/eclipse/smarthome/model/core/internal/folder/FolderObserver.java

```diff
@@ -236,7 +236,8 @@ private static void checkFile(final ModelRepository modelRepo, final File file,
             try {
                 synchronized (FolderObserver.class) {
                     if ((kind == ENTRY_CREATE || kind == ENTRY_MODIFY)) {
-                        if (parsers.contains(getExtension(file.getName()))) {
+                        // we omit parsing of hidden files possibly created by editors or operating systems
+                        if (parsers.contains(getExtension(file.getName())) && !file.getName().startsWith(".")) {
```

Comment:
Indeed this issue still exists: Before I added my change, the FolderObserver parsed my .test.items file and the items in there where added to the model and showed up in the paperui